systems/summit_user_guide.rst

```diff
@@ -2474,38 +2474,41 @@ each of its nodes.
 Debugging
 =========
 
-Arm DDT
+Linaro DDT
 -------
 
```

Comment:
Underline too short on 2478

Reply:
fixed in 15e4ea35ac8e6c884ee1e03be22a3c76c1d214d3